src/ck-pics/trace-perf.C

```diff
@@ -307,14 +311,23 @@ void TraceAutoPerf::traceClose(void)
 
 void TraceAutoPerf::printSummary() { }
 
-void TraceAutoPerf::summarizeObjectInfo(double &maxtime, double &totaltime,
+void TraceAutoPerf::summarizeObjectInfo(double &maxtime, double &totaltime, double &mintime,
     double &maxMsgCount, double &totalMsgCount, double &maxMsgSize,
-    double &totalMsgSize, double &numObjs) {
+    double &totalMsgSize, double &numObjs, double &maxBytesPerMsg, double &minBytesPerMsg) {
   void *maximum = NULL;
+
+  if(objectLoads.size() > 0) {
+    mintime = objectLoads.begin()->second->executeTime;
```

Comment:
Should this also set `maxtime` or is that guaranteed to be initialized to 0 or something?